api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
+
     # overrides ODMFilterMixin
     def get_default_odm_query(self):
         user = self.get_user()
         return (
-            Q('contributors', 'eq', user) &
+            Q('contributors', 'contains', user._id) &
+            Q('is_folder', 'ne', True) &
             Q('is_collection', 'ne', True) &
-            Q('is_deleted', 'ne', True)
+            Q('is_deleted', 'ne', True) &
+            Q('is_registration', 'ne', True)
         )
 
     # overrides ListAPIView
     def get_queryset(self):
-        current_user = self.request.user
-        if current_user.is_anonymous():
-            auth = Auth(None)
-        else:
-            auth = Auth(current_user)
```

Comment:
`NodesList.get_default_odm_query` [includes](https://github.com/CenterForOpenScience/osf.io/blob/35596266100a1c7848a936d59296bde2fb27b5a1/api/nodes/views.py#L235-L247) a permissions filter that is subtly different from the [new one](https://github.com/CenterForOpenScience/osf.io/blob/35596266100a1c7848a936d59296bde2fb27b5a1/api/users/views.py#L294-L309) in `UserNodes.get_default_odm_query`—another reason to share code between the two (cf. https://github.com/caneruguz/osf.io/pull/17#discussion_r51285543).